tools/scripts/init.cmd

```diff
@@ -16,6 +16,10 @@ if /I "%FLAVOR_DAT%"   == "Debug"   set FLAVOR_DAT=Debug
 if /I "%FLAVOR_DAT%"   == "Release" set FLAVOR_DAT=Release
 if    "%FLAVOR_DAT%"   == ""        set FLAVOR_DAT=Release
 
+if /I "%FLAVOR_WIN%"   == "Debug"   set FLAVOR_WIN=Debug
+if /I "%FLAVOR_WIN%"   == "Release" set FLAVOR_WIN=Release
+if    "%FLAVOR_WIN%"   == ""        set FLAVOR_WIN=%FLAVOR_DAT%

```

Comment:
Default should be Release NOT FLAVOR_DAT. FLAVOR_DAT is an unrelated setting and, if we change the default or that in the future it shouldn't bleed over into FLAVOR_WIN.